### ISSUE TITLE
checkbox livewire update not triggered if click is made on label

### ISSUE THREAD

**Issue by jcerdan** [livewire/flux#611]
Hi! 👋

I'm currently using a form with a checkbox. This checkbox has `wire:model.live="enabled"`
I'm watching the value of `enabled` with var_export.

this is the code:
```
         {{ var_export($form->enabled) }}
          <flux:checkbox
              wire:model.live="form.enabled"
              label="Enabled?" />
```

looking the browser console you can see that the livewire update is only triggered if you click on the checkbox square. If you click on the checkbox label (Enabled?) the livewire update is not triggered.

I'm using last Fluxui 1.0.20 and last Livewire 3.5.12

Best regards

**Comment by ju5t** [comment]
Can confirm, this issues is new in 1.0.20. I've tested it with 1.0.19 and it does send a request to the backend there.

A fully copy/pasteable version:

```php
<?php
\Livewire\Volt\state([
    'enabled' => false,
])
?>

<div class="flex p-6">
    <div class="space-x-0 space-y-3 lg:space-y-3 lg:space-x-0 lg:mt-0">
        @json($enabled)
        <flux:checkbox wire:model.live="enabled" label="Enabled?" />
    </div>
</div>
```

**Comment by RezaODB** [comment]
Also have the same problem on my app! I spent hours figuring out the issue! Thanks. A fix is needed asap please

**Comment by calebporzio** [comment]
This has been fixed and released: v1.0.21
